corehq/tabs/tabclasses.py

```diff
@@ -129,11 +130,14 @@ def view(self):
     def sidebar_items(self):
         tools = self._get_tools_items()
         report_builder_nav = self._get_report_builder_items()
+        user_management_reports = UserManagementReportDispatcher.navigation_sections(
```

Comment:
I'd imagined this being in the users section, because I think it will be used by different people than most other reports. Has this come up with the product team?

Reply:
Yes.
It was easiest to put it in reports, so i went with that because it's not clear where they would want it eventually, but that very well could be users tab.
Will move it there.
Thanks for pushing for this.

Reply:
https://github.com/dimagi/commcare-hq/pull/29909/commits/81784308f125d6b531ddfd8949dc037167f2d7e3